### ISSUE TITLE
Info: No releases on google play for now

### ISSUE THREAD

**Issue by imsodin** [syncthing/syncthing-android#2064]
It's our time again for a random denial of the access to device storage by google play. I'll appeal of course, however they will likely be slow to respond and have stupid requirements, so it will take a while.

(Just as we got more contributions, which I am already "struggling" to keep up with - more like not keeping up with. If anyone affected reads this - sorry about that, I will get to them. Thanks a lot for contributing!)

**Comment by imsodin** [comment]
Unfortunately no progress. Few times back and forth with them, but no real interaction yet. As in I give targeted and concrete explanations using their own criteria and wording to explain why we need this permissions and fulfill the requirements to be allowed to use it. And ask them to write what's wrong about that or give any concrete suggestions. And then I receive another boilerplate email not engaging with anything I wrote. I'll continue that game in the hopes that something eventually randomly changes, but there's no indication that things will improve at this time.

**Comment by gapey-bulk** [comment]
_I registered solely for commenting this._
So, I wanted to ask -- may we get release on F-Droid please? There's also just 1.27.3 sitting there.

**Comment by bt90** [comment]
F-Droid updates tend to be a bit slow. However, I'll investigate if there's a problem with the build.

**Comment by imsodin** [comment]
There aren't any problems on their side, it's on mine - I didn't do 1.27.4. At the time I was foolish enough to hope for a quick-ish resolution of the google play situation and was thus pausing releases. 1.27.5 will happen soon(.TM).

**Comment by Nimras186** [comment]
Any idea when Android will start seeing updates again still on 1.27.3?

**Comment by ThirtySomething** [comment]
I'm a developer, too. But I've got no experience with Android developent. Is there a way to support you somehow? I'm not sure but I assume that with this new permission at least one [issue](https://forum.syncthing.net/t/android-two-way-synchronization/21890) might be fixed.

**Comment by imsodin** [comment]
Update: No update. The last one still applies: https://github.com/syncthing/syncthing-android/issues/2064#issuecomment-2015948576. Except that I wont play the game of pointless email ping-pong anymore. I'll just keep releasing and they'll probably keep rejecting. If you can, switch to F-Droid. I don't know yet what I will do mid-term with the situation. Of course I'll drop an update here if anything new comes up.

Also there's no way to help I am aware of. Unless you are in the very unlikely position to change google play's position from the inside :) Please do not use this issue to try to organise some attention thing or whatever (if you want to do something you can of course, not here though, and I doubt it helps).

**Comment by Nimras186** [comment]
> Update: No update. The last one still applies: [#2064 (comment)](https://github.com/syncthing/syncthing-android/issues/2064#issuecomment-2015948576). Except that I wont play the game of pointless email ping-pong anymore. I'll just keep releasing and they'll probably keep rejecting. If you can, switch to F-Droid (and everyone on android 11 or so can, since then there are no more manual updates needed so except for the initial install F-Droid really isn't a hassle to use anymore on plain stock android). I don't know yet what I will do mid-term with the situation. Of course I'll drop an update here if anything new comes up.
> 
> Also there's no way to help I am aware of. Unless you are in the very unlikely position to change google play's position from the inside :) Please do not use this issue to try to organise some attention thing or whatever (if you want to do something you can of course, not here though, and I doubt it helps).

I'm sorry to hear that there seems to be no end in sight from google.
In regards to F-Droid I have no clue what that is or how that work, I might be able to figure it out but not sure if my friend who uses it 2 will as I set it up last time I was in Denmark very not technical not sure I can explain how to use it.
But I will try and remember to look up this F-Droid thing just wrong we have to go this far to get something updated because Google are being sad about it.

**Comment by minzdrav** [comment]
> Update: No update. The last one still applies: https://github.com/syncthing/syncthing-android/issues/2064#issuecomment-2015948576. Except that I wont play the game of pointless email ping-pong anymore. I'll just keep releasing and they'll probably keep rejecting. If you can, switch to F-Droid (and everyone on android 11 or so can, since then there are no more manual updates needed so except for the initial install F-Droid really isn't a hassle to use anymore on plain stock android). I don't know yet what I will do mid-term with the situation. Of course I'll drop an update here if anything new comes up.
> 
> Also there's no way to help I am aware of. Unless you are in the very unlikely position to change google play's position from the inside :) Please do not use this issue to try to organise some attention thing or whatever (if you want to do something you can of course, not here though, and I doubt it helps).

Hi @imsodin 
Thank you for the update!
What is the recommended way to switch from Google Play to F-Droid? In the F-Droid UI I can see only the delete button. Looks like it's not possible to update the Google Play version from the F-Droid. Should I remove and reinstall? What about configuration? Do I need to set up all folders again?

**Comment by Nimras186** [comment]
> Installing F-Droid is not that hard, a few initial manual steps and then it's all automatic:
> 
>     1. Install F-Droid - requires setting some unknown app permissions, see e.g. these instructions: https://www.popsci.com/diy/f-droid-download/#h-how-to-set-up-f-droid
> 
>     2. Open F-Droid, tap the search icon towards the bottom right to search for syncthing, tap it and install it (confirm).
> 
>     3. Check automatic updates is enabled (not sure if it's the default): Bottom right is the settings symbol, look for "Automatically install updates".
> 
>     4. Be happy or something.
> 
> 
> The synthing setup stuff is of course a bit more annoying. There's the config backup/import functionality that can help with that.
> 
> If you have any questions on that, please ask on the forum - https://forum.syncthing.net. Generally to ask about this issue or discuss it, please use the forum not this issue. People might subscribe to it to get relevant updates (and most devs are subscribed by default), so lots of people would be notified without any actual update.

Thank you so much for your answer, I will look into it when I get home from my holiday next week, hopefully at some point you get it into Google thick head that you need what ever it is they block you on so we can get our updates back. Thank you for your amazing work I will keep my fingers crossed.

And off course if something change so I might be able to help as a normal user with no extra skills to get google to play nice just let me know if I am able to do it I will.

**Comment by KarlM256** [comment]
Hi,

Thanks for all your hard work to support Syncthing.

Would you mind keeping builds here, signed the way you do now? That way we can test release candidates and provide feedback. It would also allow folks that have everything set up for the existing Google Play configuration to simply download the apk from GitHub and upgrade it.

Thanks,

Karl

**Comment by imsodin** [comment]
> Would you mind keeping builds here, signed the way you do now? That way we can test release candidates and provide feedback. It would also allow folks that have everything set up for the existing Google Play configuration to simply download the apk from GitHub and upgrade it.

Yes the exact same apks as now will also be available on the github releases for now. I am not entirely sure those are a drop-in for the Google Play install though - I think Google Play forced some app package thing on us a while ago, that might have changed how they are signed.

**Comment by KarlM256** [comment]
Hi,

I have been doing exactly that for 1.27.4 5 and 6. Of course, not to say
that won't change.

Thanks,

Karl

**Comment by imsodin** [comment]
I am still on this.  
One action taken was to temporarily unlist the app, and it looks like it's still unlisted at the time of writing even though I reverted the change. Anyway in my book it's a good thing that no new users install an outdated app where the future of the distribution channel (gplay) is unclear. It might become listed again soon with or without the permission granted (certainly if it is granted).  
I am still in communication with google play, and there's been an exchange of messages where I actually felt some engagement. The situation is still not clear nor is there any timeline, but "something" is ongoing.

**Comment by rogers1106** [comment]
Are releases now limited to F-Droid? Nothing on GitHub anymore? I'm missing 1.27.6.2 here.

**Comment by tomasz1986** [comment]
> Are releases now limited to F-Droid? Nothing on GitHub anymore? I'm missing 1.27.6.2 here.

They are available at https://github.com/syncthing/syncthing-android/releases. Please note that there is no actual difference between v1.27.6 and v1.27.6.2, and also it is true that there has been no new release for quite a while, as Syncthing proper is currently at v1.27.10 already.

**Comment by imsodin** [comment]
What @tomasz1986 said. I also will do a release soon-ish (as usual, F-Droid and github), some internal cleanup to be taking care of. No updates on the google play front, except that "things" are still ongoing and the situation is still unclear.

**Comment by rogers1106** [comment]
Thqnkos for the release. 😀

**Comment by Alfa21tpp** [comment]
Hi @imsodin , I doubt they will ever tell you which part of the code generates the problem on GP.
Perhaps a better choice is to compare the latest accepted version of the code with the first one that is no longer accepted.
Another way perhaps is to compare this project with the forked one by Catfriend1 which seems not to suffer from update problems.
I know mine are just words... but I hope to shine some light or ideas. Sometimes when dealing with black boxes it's better to make changes instead of asking the boxed heads why ;-)

**Comment by imsodin** [comment]
tl;dr: The app is discontinued, so naturally also no google play releases - see readme.

Nothing came of the discussions with google. Demands by Google for changes to get the permission granted were vague, which makes it both arduous to figure out how to address them and very unclear if whatever I do will actually lead to success. Then more unrelated work to stay on play came up (dev. verification, target API level), which among other influences finally made me realize I don't have the motivation or time anymore to play this game.
